package.json

```diff
@@ -63,6 +64,16 @@
       "@commitlint/config-conventional"
     ]
   },
+  "jest": {
```

Comment:
I don't think we need to configure jest. We only need the `--coverage` flag in the npm script. I might be mistaken but I think Jest already ignores any files present in gitignore.

Reply:
We need it since it is trying to run mocks as tests inside `__tests__` folder.
Although I think we just need testRegex and not the other properties. Or we could change the name of the `__tests__` folders.